### ISSUE TITLE
About 6d representation 

### ISSUE THREAD

**Issue by gen4u** [Jiyao06/GenPose#28]
Thanks for your great work.

I have a question about training the rotation matrix in the ScoreNet model. 
According [1], for rotation matrix 6d representation is used, but when I look at the Genpose training code, it seems that the loss is applied in the 6d representation space, not in the original space. Is there a significant difference between applying loss in these two ways?

[1] Yi Zhou, Connelly Barnes, Jingwan Lu, Jimei Yang, and Hao Li. On the continuity of rotation representations in neural networks. In Proceedings of the IEEE/CVF Conference on Computer Vision and Pattern Recognition, pages 5745–5753, 2019.

**Comment by Jiyao06** [comment]
Thank you for your suggestion. We apologize for not conducting such a comparative experiment. I believe it is necessary, and we will consider running this comparison.

**Comment by Jiyao06** [comment]
We did not supervise in 9D because we are supervising the score of the 6D rotation, not the ground truth 6D rotation itself. We are not performing a strict diffusion process in the SO(3) manifold.
